### DISCUSSION TITLE
[Feature] Add git commit message to [VSCode, Jetbrains, ...etc] git panel

### DISCUSSION THREAD

**Discussion by ZorinFoss** [pieces-app/support#256] [General]
Hi pieces team,

It will be very helpful when pieces help user with writing git commit message by monitor `git diff` result. 

Thanks,

**Comment by arthur-t-w-mills** [comment]
Love this idea. This would be extremely handy, as Pieces could even use the LTM to get all activity related to that Git branch. Super keen on this 🙂
